desktop-widgets/mainwindow.cpp

```diff
@@ -297,8 +297,6 @@ MainWindow::MainWindow() : QMainWindow(),
 
 	// now let's set up some connections
 	connect(graphics, &ProfileWidget2::enableToolbar ,this, &MainWindow::setEnabledToolbar);
```

Comment:
typo in the first commit message. Look for 'toe' :-)

Reply:
fixed.